clash-prelude/src/Clash/Tutorial.hs

```diff
@@ -1271,10 +1288,18 @@ a general listing of the available template holes:
 * @~ISLIT[N]@: Is the @(N+1)@'th argument to the function a literal.
 * @~ISVAR[N]@: Is the @(N+1)@'th argument to the function explicitly not a
   literal
-* @~ISGATED[N]@: Is the @(N+1)@'th argument a gated clock, errors when called on
-  an argument which is not a 'Clock'.
-* @~ISSYNC[N]@: Is the @(N+1)@'th argument a synchronous reset, errors when
-  called on an argument which is not a 'Reset'.
+* @~TAG[N]@: Name of given domain. Errors when called on an argument which is not
+  a 'KnownDomain'.
+* @~PERIOD[N]@: Clock period of given domain. Errors when called on an argument
+  which is not a 'KnownDomain'.
+* @~ISENABLED[N]@: Is the @(N+1)@'th argument a an Enable line set to a constant
+  True. Errors when called on an argument which is not an 'Enable'.
+* @~ISSYNC[N]@: Does synthesis domain of the @(N+1)@'th have synchronous resets. Errors
+  when called on an argument which is not a 'KnownDomain'.
+* @~ISINITDEFINED[N]@: Does synthesis domain of the @(N+1)@'th have defined initial
+  values. Errors when called on an argument which is not a 'KnownDomain'.
+* @~ISRISINGEDGE[N]@: Does synthesis domain of the @(N+1)@'th respond to rising
+  edges. Errors when called on an argument which is not a 'KnownDomain'.
```

Comment:
Similar to `~ISENABLED`,  `~ISRISINGEDGE` has a weird name particularly because you have to use in on a KnownDomain.

Reply:
Fixed in https://github.com/clash-lang/clash-compiler/pull/527/commits/4b01f6f8b23bf815c531abe32734b4c99fb299f8